src/components/page-list.jsx

```diff
@@ -126,3 +125,11 @@ function debounce (func, delay) {
     timer = setTimeout(() => func(...args), delay);
   };
 }
+
+function formatSites (tags) {
+  const isSite = /^site:/;
+  return tags
+    .filter(tagging => isSite.test(tagging.name))
+    .map(tagging => tagging.name.replace(isSite, ''))
```

Comment:
Yea, I was a bit confused because the server and app use 2 different module styles so didn't think it possible, but after some reading, realized I could go with CommonJS to get it to work. (There is some Babel magic happening there that I don't quite understand.)

I changed `formatters.js` to use `exports.foo` instead of `export foo`. This gave me linting errors about `exports not being defined`, so I added `/*eslint-env node*/`. Unsure if that's the right way to treat that error? Could use some help here 😕

Reply:
👍 I think you  probably want `/* eslint-env commonjs */`, but `node` works fine, too.

Reply:
(But otherwise, yeah, you’re doing the right thing! The alternative is to edit `src/.eslintrc.json`, but I think we want to keep this to just the files where it’s needed, so I would have done the same as you.)